sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Change Log azure-data-appconfiguration
+
+## Version 1.0.0-preview.2:
+
+For details on the Azure SDK for Java (August 2019 Preview) release refer to the [release announcement](https://aka.ms/azure-sdk-preview2-java).
+
+- Merged ConfigurationClientBuilder and ConfigurationAsyncClientBuilder into ConfigurationClientBuilder. Method to build each client were added
```

Comment:
Should end bulleted lists with periods as part of the style guide.
https://docs.microsoft.com/en-ca/style-guide/punctuation/periods

Reply:
Added periods